mustache.js

```diff
@@ -420,6 +426,19 @@
           }
         } else {
           intermediateValue = context.view[name];
+
+          /**
+           * Only checking against hasProperty, which always returns false if
```

Comment:
Well, I'll add them eventually, before merge. Feel free to beat me to it!

I also like using triple backtick + js + triple backtick (same as in markdown) to wrap examples.